### PR TITLE
Updating recipes

### PR DESCRIPTION
**Pull request by BusbyActual**
Progress #1662

- [x] [JSPM and SystemJS for ES2015](https://github.com/avajs/ava/blob/master/docs/recipes/jspm-systemjs.md)

### CODE REVIEW

**Comment by novemberborn** [comment]
@forresst were you making suggestions? They don't seem to have come through.

@BusbyActual been a bit busy lately, so I haven't had a chance yet to look at your changes here. Thanks for your work on this though!

**Comment by BusbyActual** [comment]
@novemberborn there was a comment on the French translation I updated.

**Comment by novemberborn** [comment]
@BusbyActual yes I saw that later, thanks 👍

**Comment by forresst** [comment]
@BusbyActual @novemberborn sorry for the inconvenience

File: docs/recipes/jspm-systemjs.md [hunk 1]
```diff
@@ -14,11 +11,15 @@ This recipe has only been tested with JSPM v0.17.0-beta.22, but it should work w
 
 ### Babel
 
-Configure your .babelrc to work with AVA if you have not already. NOTE: You can keep additional configuration in your JSPM config files to override these settings during bundling and building.
+Configure your .babelrc to work with AVA if you have not already. NOTE: You can keep additional configuration in your JSPM config files to override these settings during bundling and building. As of babel 7 [@babel/preset-env](https://www.npmjs.com/package/@babel/preset-env) will transform code as babel-preset-esXXXX did previously.
```
Comment (novemberborn) [inline docs/recipes/jspm-systemjs.md]: I don't think we need to explain the virtues of `@babel/preset-env`.

File: docs/recipes/jspm-systemjs.md [hunk 2]
```diff
@@ -1,6 +1,3 @@
-> **Please note, this recipe has not yet been updated for Babel 7 support in AVA 1.0.**
-
----
```
Comment (novemberborn) [inline docs/recipes/jspm-systemjs.md]: This line ought to be removed as well.

File: docs/recipes/jspm-systemjs.md (hunk 1 above)
Comment (forresst) [reply docs/recipes/jspm-systemjs.md]: @novemberborn Personally, I find the indication useful. It allows to know that the behavior is the same.

Comment (novemberborn) [reply docs/recipes/jspm-systemjs.md]: Yes but IMHO that's part of Babel 7's upgrade documentation, not AVA's.

Comment (forresst) [reply docs/recipes/jspm-systemjs.md]: OK. I agree with you

File: docs/recipes/jspm-systemjs.md (hunk 2 above)
Comment (BusbyActual) [reply docs/recipes/jspm-systemjs.md]: Sounds good to me! I've addressed the requested changes : ^)

**Comment by novemberborn** [comment]
@BusbyActual I've pushed a few tweaks, let me know if you're happy with them.

Looking at this recipe I have no idea why it even mentions Babel, but that's a worry for another day I suppose.

**Pull request merged**